seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/ConnectorSpecificationCheckTest.java

```diff
@@ -152,16 +154,29 @@ public void testAllConnectorImplementFactoryWithUpToDateMethod() throws ClassNot
                 log.info(
                         "Check sink connector {} successfully", factory.getClass().getSimpleName());
 
-                checkSupportMultiTableSink(sinkClass);
+                checkSupportMultiTableSink(factory, sinkClass);
             }
         }
     }
 
-    private void checkSupportMultiTableSink(Class<? extends SeaTunnelSink> sinkClass) {
+    private void checkSupportMultiTableSink(
+            TableSinkFactory sinkFactory, Class<? extends SeaTunnelSink> sinkClass) {
         if (!SupportMultiTableSink.class.isAssignableFrom(sinkClass)) {
             return;
         }
 
+        OptionRule sinkOptionRule = sinkFactory.optionRule();
+        Assertions.assertTrue(
+                sinkOptionRule
+                                .getOptionalOptions()
+                                .contains(SinkCommonOptions.MULTI_TABLE_SINK_REPLICA)
+                        || sinkOptionRule
+                                .getRequiredOptions()
+                                .contains(SinkCommonOptions.MULTI_TABLE_SINK_REPLICA),
+                "Please add `SinkCommonOptions.MULTI_TABLE_SINK_REPLICA` option into the `optionRule` method of `"
+                        + sinkFactory.getClass().getSimpleName()
+                        + "`");
```

Comment:
MULTI_TABLE_SINK_REPLICA should be optional options, can not be required options.